src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -252,6 +252,14 @@ void EIRPlantLoopHeatPump::doPhysics(EnergyPlusData &state, Real64 currentLoad)
         return;
     }
 
+    Real64 currentLoadConsolidated = currentLoad;
+    this->oriLoopLoad = currentLoad;
+    if (this->validConcurrentRunFlag && this->consolidateLoadsFlag == true) {
+        currentLoadConsolidated = abs(this->oriLoopLoad) > abs(this->companionHeatPumpCoil->oriLoopLoad)
+                                      ? this->oriLoopLoad
+                                      : (-1.0) * this->companionHeatPumpCoil->oriLoopLoad;
+    }
```

Comment:
Isn't the logic here that if this load is greater than the companion load then this HP will operate (i.e., running = true)? Otherwise this HP is off. And this only works if there is actually a companion HP. This logic assumes there is always a companion HP which may not be true.